lang/en/docs/workspaces.md

```diff
@@ -0,0 +1,92 @@
+---
+id: docs_workspaces
+guide: docs_workspaces
+layout: guide
+---
+
+{% include vars.html %}
+
+Workspace are a new way to setup your package architecture that's available by default starting from Yarn 1.0. It allows you to setup multiple packages in such a way that you only need to run `yarn install` once to install all of them in a single pass.
```

Comment:
s/Workspace/Workspaces

Reply:
>you only need to run `yarn install` once to install all of them in a single pass

you only need to run `yarn install` at the root, to install everything in a single pass.

Reply:
they can be installed from anywhere, actually, as long as you're in the workspace tree